pages.zh/android/input.md

```diff
@@ -0,0 +1,25 @@
+# input
+
+> 将事件代码或触摸屏手势发送到 Android 设备。
+> 此命令只能通过`adb shell`使用。
```

Comment:
I think spaces should be between all mixed English/Chinese, even if the English is in backticks, correct?
```suggestion
> 此命令只能通过 `adb shell` 使用。
```

Reply:
Yes, we can unify the format specification.